lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller.pm

```diff
@@ -16,96 +16,9 @@
-    my $files
-      = Mojo::File->new($self->_home)->list({dir => 1})->grep(sub { -d $_ })->map('basename')
-      ->grep(qr/^(?!test|WebAPIPlugin|__pycache__)/)->to_array;
-
-    $self->render('ObsRsync_index', folders => $files);
-}
-
-sub folder {
-    my $self   = shift;
-    my $folder = $self->param('folder');
-    return undef if $self->_check_and_render_error($folder);
-
-    my $full        = $self->_home;
-    my $obs_project = $folder;
-    my $files       = Mojo::File->new($full, $obs_project, '.run_last')->list({dir => 1})->map('basename');
-
-    $self->render(
-        'ObsRsync_folder',
-        obs_project     => $obs_project,
-        lst_files       => $files->grep(qr/files_.*\.lst/)->to_array,
-        read_files_sh   => $files->grep(qr/read_files\.sh/)->join(),
-        rsync_commands  => $files->grep(qr/rsync_.*\.cmd/)->to_array,
-        rsync_sh        => $files->grep(qr/print_rsync.*\.sh/)->to_array,
-        openqa_commands => $files->grep(qr/openqa\.cmd/)->to_array,
-        openqa_sh       => $files->grep(qr/print_openqa\.sh/)->join());
-}
-
-sub logs {
-    my $self   = shift;
-    my $folder = $self->param('folder');
-    return undef if $self->_check_and_render_error($folder);
-
-    my $full = Mojo::File->new($self->_home, $folder);
-    my $files
-      = $full->list({dir => 1, hidden => 1})->map('basename')->grep(qr/\.run_.*/)->sort(sub { $b cmp $a })->to_array;
-    $self->render('ObsRsync_logs', folder => $folder, full => $full->to_string, subfolders => $files);
-}
-
-sub logfiles {
-    my $self      = shift;
-    my $folder    = $self->param('folder');
-    my $subfolder = $self->param('subfolder');
-    return undef if $self->_check_and_render_error($folder, $subfolder);
-
-    my $full  = Mojo::File->new($self->_home, $folder, $subfolder);
-    my $files = $full->list({dir => 1})->map('basename')->sort->to_array;
-    $self->render(
-        'ObsRsync_logfiles',
-        folder    => $folder,
-        subfolder => $subfolder,
-        full      => $full->to_string,
-        files     => $files
-    );
-}
-
-sub download_file {
-    my $self      = shift;
-    my $folder    = $self->param('folder');
-    my $subfolder = $self->param('subfolder');
-    my $filename  = $self->param('filename');
-    return undef if $self->_check_and_render_error($folder, $subfolder, $filename);
-
-    my $full = Mojo::File->new($self->_home, $folder, $subfolder);
-    $self->reply->file($full->child($filename));
-}
-
-sub run {
-    my $self   = shift;
-    my $folder = $self->param('folder');
-    return undef if $self->_check_and_render_error($folder);
-
-    my @args = ($self->_home . "/rsync.sh", $folder);
-
-    my $out;
-    eval { $out = system([0], "bash", @args); 1 }
-      or return $self->render(json => {output => $@, code => $EXITVAL}, status => 500);
-
-    return $self->render(json => {output => $out, code => $EXITVAL}, status => 201);
+    return shift->obs_rsync->home;
 }
```

Comment:
This method does not appear to serve much of a purpose now that the helper exists.